.cirrus.yml

```diff
@@ -1,3 +1,29 @@
+task:
+  auto_cancellation: true
+
+  env:
+      PY_VER: "3.6"
+
+  name: Black Formatting
+  container:
+      image: python:latest
+  env:
+      PATH: $HOME/conda/bin:$PATH
+  conda_script:
+      - curl https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh > install.sh
+      - bash install.sh -b -p $HOME/conda
+      - conda update -yn base conda
+      - conda install -y python=$PY_VER
+      - rm install.sh
+
+  install_script:
+    # pip >= 19.0 crashes on Linux with Python 3.7
+    - python -m pip install --retries 3 --upgrade "pip<19.0"
+    - pip install --retries 3 -r requirements.txt
+
+  test_script:
+    - black -S -l 79 -t py36 --exclude vendored\|_vispy --check .
```

Comment:
shouldn't all of these command line options (except `--check`) already be set by the `pyproject.toml` (and therefore autorun when we `black --check`)?

Reply:
nope, those are for pre-commit

Reply:
https://github.com/python/black#pyprojecttoml

Reply:
yeah but we have excludes and stuff, also that line matches our contributing guideline?

Reply:
our contributing guidelines are there to tell people what it's equivalent to but i would prefer to cut down on redundancy wherever possible